MetaMask/metamask-extension
# ONBOARDING_V2: fix Privacy Policy link in Spanish

## Explanation

Fixes #16837

Fixes Privacy Policy link on ONBOARDING_V2 Advanced Configuration page

## Screenshots/Screencaps

### Before

See #16837



### After

<img width="643" alt="Screen Shot 2022-12-13 at 04 13 20" src="https://user-images.githubusercontent.com/20778143/207156334-21d39eb8-1afd-411c-a717-73b7c4f02588.png">


## Manual Testing Steps

1. Enable ONBOARDING_V2
2. Load MM for the first time
3. Go through the Onboarding Process
4. On the Privacy Settings page, set the Spanish language
5. Observe Privacy Policy link is absent

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
